forum/users/email.py

```diff
@@ -0,0 +1,12 @@
+from djoser import email
+from django.contrib.sites.shortcuts import get_current_site
+
+
+class PasswordResetEmail(email.PasswordResetEmail):
+    template_name = "email/password_reset.html"
+
+    def get_context_data(self):
+        context = super().get_context_data()
+        context["domain"] = get_current_site(self.context["request"]).domain
```

Comment:
Using `get_current_site` works well for dynamic domain detection. However, in production, it is often cleaner and more secure to explicitly define the domain in environment variables.
In `settings.py`:
```
DOMAIN_NAME = os.getenv("DOMAIN_NAME", "localhost")
```